detection_rules/packaging.py

```diff
@@ -461,6 +472,38 @@ def generate_xslx(self, path):
         doc.populate()
         doc.close()
 
+    def _generate_registry_package(self, save_dir):
+        """Generate the artifact for the oob package-storage."""
+        from .schemas.registry_package import RegistryPackageManifest
+
+        assert self.registry_data
+
+        manifest = RegistryPackageManifest.from_dict(self.registry_data)
+
+        package_dir = Path(save_dir).joinpath(manifest.version)
+        docs_dir = package_dir / 'docs'
+        rules_dir = package_dir / 'kibana' / 'rules'
+
+        docs_dir.mkdir(parents=True)
+        rules_dir.mkdir(parents=True)
```

Comment:
Do you want an error to be raised if these directories already exist?

```suggestion
        docs_dir.mkdir(parents=True, exist_ok=True)
        rules_dir.mkdir(parents=True, exist_ok=True)
```